### PR TITLE
Create 2018-08-14.md

### PR DESCRIPTION
**Pull request by jridgewell**

### CODE REVIEW

File: meetings/2018-08-14.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...

+    * RB: A lot of cases we’ve done actually replace the constructor.

+    * DFV: Not including decorator will break reifing later.

+    * DH: Same lexical environment won’t always allow you to re-eval. The decorator could be non-pure, etc.

+    * RB: If we do include, why are we not including export?

+    * JH: It modiefies the module, not the value. You can’t introspect the value to determine if it’s exported.
```
Comment (kt3k) [inline meetings/2018-08-14.md:23]: modiefies -> modifies?

File: meetings/2018-08-14.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...

+    * RC: that’s just the same as a visibility modifier

+    * BG: Angular has had a ton of users who’ve had no issues with this.

+    * RC: export changes the shape of the module?

+* Defensive Methods

+    * JH: I undetand why we want to prevent leaks, but I don’t understand why we want to prevent communication channels.
```
Comment (kt3k) [inline meetings/2018-08-14.md:43]: undetand -> understand?

File: meetings/2018-08-14.md (hunk 1 above)
Comment (jridgewell) [reply meetings/2018-08-14.md:23]: @kt3k: would you submit a pull request to fix the mispellings?

Comment (kt3k) [reply meetings/2018-08-14.md:23]: Yes! #142

**Pull request merged**